contribs/entropy2mnemonic/README.md

```diff
@@ -0,0 +1,37 @@
+# entropy2mnemonic
+
+Standalone version of `gnokey add --entropy` for generating BIP39 mnemonics from custom entropy. Useful for creating mnemonics for hardware wallets like Ledger or any situation where you want deterministic key generation from your own entropy source.
+
+Uses the same entropy-to-mnemonic conversion as gnokey: SHA-256 hash of your input entropy is used as the seed for BIP39 mnemonic generation.
+
+## Example
+
+```bash
+$ entropy2mnemonic
+=== ENTROPY TO MNEMONIC CONVERTER ===
+
+This tool generates a BIP39 mnemonic from your custom entropy.
+The same entropy will always produce the same mnemonic.
+
+REQUIREMENTS:
```

Comment:
This is too complicated. 

Everything under REQUIREMENTS and GOOD ENTROPY SOURCES to be entirely replaced with the following:

INSTRUCTIONS:
Exactly 42 rolls of D20 dice (example)
Exactly 52 cards, 20 times shuffled (example)

Nothing more, nothing less.

Reply:
[`1086972` (#4586)](https://github.com/gnolang/gno/pull/4586/commits/1086972d8e9c99417e0d56c2bedeabe470d31aa6)